src/BotPlutusInterface/ChainIndex.hs

```diff
@@ -105,24 +102,16 @@ chainIndexQueryOne pabConf endpoint = do
       | otherwise -> error (show failureResp)
     Left failureResp -> error (show failureResp)
 
--- | Query for utxo's and filter collateral utxo from result.
+-- | Query for utxo's.
 chainIndexUtxoQuery :: forall (w :: Type). ContractEnvironment w -> ClientM UtxosResponse -> IO UtxosResponse
 chainIndexUtxoQuery contractEnv query = do
-  collateralUtxo <- readCollateralUtxo contractEnv
-  let removeCollateral :: UtxosResponse -> UtxosResponse
-      removeCollateral (UtxosResponse tip page) = UtxosResponse tip (removeCollateralFromPage collateralUtxo page)
-  removeCollateral
-    <$> chainIndexQueryMany
-      contractEnv.cePABConfig
-      query
+  chainIndexQueryMany
+    contractEnv.cePABConfig
+    query
 
--- | Query for txo's and filter collateral txo from result.
+-- | Query for txo's.
 chainIndexTxoQuery :: forall (w :: Type). ContractEnvironment w -> ClientM TxosResponse -> IO TxosResponse
 chainIndexTxoQuery contractEnv query = do
-  collateralUtxo <- readCollateralUtxo contractEnv
-  let removeCollateral :: TxosResponse -> TxosResponse
-      removeCollateral (TxosResponse page) = TxosResponse (removeCollateralFromPage collateralUtxo page)
-  removeCollateral
```

Comment:
Removing now handled by `withColalteralHandling` in `handlePABEffect` and `runPABEffectPure`